api/SVGAnimateColorElement.json

```diff
@@ -14,19 +14,19 @@
             "version_added": null
```

Comment:
`true`

Reply:
I come up with `false` for all flavors of Chrome. What makes you think it's `true`.

Reply:
Because `opera` + `opera_android` says `true`, but I wasn’t entirely sure about it.

Reply:
Chrome does not seem to support animateColor. Here's a testcase: https://bugzilla.mozilla.org/attachment.cgi?id=322937

Reply:
Opera is just Chrome with a different skin these days. It will always match.

Reply:
You're generally right, but there are exceptions.